presto-main/src/main/java/com/facebook/presto/metadata/Metadata.java

```diff
@@ -315,6 +316,16 @@
      */
     List<GrantInfo> listTablePrivileges(Session session, QualifiedTablePrefix prefix);
 
+    /**
+     * @return whether writing staging files is enabled
```

Comment:
nit: Looks like we don't put `@return`. but just say `Returns whether writing staging files is enabled`

Reply:
I don't think the javadoc style is used consistently in this interface. Both styles (using `@return` or not) both exist.